jnigen/test/kotlin_test/kotlin/src/main/kotlin/com/github/dart_lang/jnigen/SuspendFun.kt

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+ * for details. All rights reserved. Use of this source code is governed by a
```

Comment:
I believe we should give this a more specific directory name than kotlin_test, we likely want to support more Kotlin features later.

Reply:
It will be similar to simple package test that tests multiple things. I'll keep on adding classes for other types of functionality to kotlin_test.